Projet-Web-Avance-A4/Developpeur
# style(download) : add style to download component page

**How to Test ?** 

1) Go to this [page](http://localhost:3000/download-components)
2) Check if style is applied. Should look like that : 
![image](https://github.com/Projet-Web-Avance-A4/Developpeur/assets/141679077/adf7cf13-59bc-4381-859d-3b571dd25b98)
